spec/factories/document_type_schema_factory.rb

```diff
@@ -5,6 +5,46 @@
     supertype { "not-sure" }
     id { SecureRandom.hex(4) }
     document_type { SecureRandom.alphanumeric(8) }
-    initialize_with { DocumentTypeSchema.create(attributes.stringify_keys) }
+    contents { [] }
+    associations { [] }
+
+    transient do
+      translations do
```

Comment:
What does this do? It rings a lot of alarm bells that we have to do this.

Reply:
This used to make up for the fact that guidance was enforced, but the factory-generated schemas had none. It was the extra bit of dynamic config to make format-specific translations work in test, but it's no longer needed.